.github/workflows/wasmcloud.yml

```diff
@@ -655,3 +669,44 @@ jobs:
         continue-on-error: ${{ github.repository_owner != 'wasmCloud' }}
         run: cargo publish --token ${{ secrets.CRATES_PUBLISH_TOKEN }}
         working-directory: ./crates/${{ matrix.crate }}
+
+  smart-release:
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          cache: false
+      - name: Install cmake for smart-release
+        run: sudo apt update && sudo apt install cmake -y
+
+      - name: install smart-release
+        env:
+          RUSTFLAGS: ''
+        run: cargo install cargo-smart-release --git https://github.com/brooksmtownsend/cargo-smart-release --branch feat/update-workspace-dependencies
```

Comment:
Could you put in why we're using your branch version here? is that normal or was this just for testing?

Reply:
Workspace dependency changes. It is in the PR description!

Reply:
Oh yeah sorry a bit late here, but I was hoping to have that explanation *in the code*, appreciate the addition!